_data/changelogs/radio-buttons.yml

```
@@ -19,4 +19,110 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+- date: 2022-08-05
+  summary: Updated aria-disabled styles to match disabled
+  summaryAdditional: Now disabled styling is applied whether you use disabled (disabled and hidden from screen readers) or aria-disabled (disabled and visible to screen readers).
+  isBreaking:
+  affectsAccessibility:
+  affectsAssets:
+  affectsGuidance:
+  affectsJavascript:
+  affectsMarkup:
+  affectsStyles:
+  githubPr: 4783
+  githubRepo: uswds
+  versionUswds: 3.1.0
+- date: 2022-04-28
+  summary: Released USWDS 3.0.
+  summaryAdditional: USWDS 3.0 allows teams to update their projects to modern Sass syntax, improve performance, and reduce the size of project CSS.
+  isBreaking: true
+  affectsAccessibility: true
+  affectsAssets: true
+  affectsGuidance: true
+  affectsJavascript: true
+  affectsMarkup: true
+  affectsStyles: true
+  githubPr: 4656
+  githubRepo: uswds
+  versionUswds: 3.0.0
+- date: 2022-04-11
+  summary: Added support for High Contrast Mode and forced colors
+  summaryAdditional: All our components now support proper display when users have a forced colors mode set in their operating system.
+  isBreaking: false
+  affectsAccessibility: true
+  affectsAssets: false
+  affectsGuidance: false
+  affectsJavascript: false
+  affectsMarkup: false
+  affectsStyles: true
+  githubPr: 4610
+  githubRepo: uswds
+  versionUswds: 2.13.3
+- date: 2021-08-18
+  summary: Improved whitespace sensitivity of radio and checkbox tiles
+  summaryAdditional: Now radio and checkbox tiles will display consistently whether or not there's extra whitespace in the markup.
+  isBreaking: false
+  affectsAccessibility: false
+  affectsAssets: false
+  affectsGuidance: false
+  affectsJavascript: false
+  affectsMarkup: false
+  affectsStyles: true
+  githubPr: 4286
+  githubRepo: uswds
+  versionUswds: 2.12.1
+- date: 2021-08-18
+  summary: Improved class order sensitivity for breadcrumb and checkbox/radio
+  summaryAdditional: Now breadcrumb and checkbox/radio components display properly regardless of the order of the class and modifier names.
```

I didn't see that this actually affected breadcrumbs, so it should be safe to remove?

```suggestion
  summary: Improved class order sensitivity for checkbox and radio.
  summaryAdditional: Now checkbox and radio components display properly regardless of the order of the class and modifier names.
```